apps/ios/Shared/Model/SimpleXAPI.swift

```diff
@@ -702,6 +702,12 @@ func apiRemoveMember(_ groupId: Int64, _ memberId: Int64) async throws -> GroupM
     throw r
 }
 
+func apiMemberRole(_ groupId: Int64, _ memberId: Int64, _ memberRole: GroupMemberRole) async throws -> GroupMember {
```

Comment:
There's api to change own role, but the option seems to be unavailable in ui - maybe we should allow to open self GroupMemberInfoView?

Reply:
I thought about it, but not critical - maybe later.